cpp/locality/Voronoi.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2010-2019 The Regents of the University of Michigan
+// This file is from the freud project, released under the BSD 3-Clause License.
+
+#ifndef VORONOI_H
+#define VORONOI_H
+
+#include "VectorMath.h"
+#include <vector>
```

Comment:
It looks like this is working right now because you have it in your CPP file, but you should #include NeighborList directly in this file.